src/server/dash-server.ts

```diff
@@ -34,7 +31,7 @@ export class DashServer {
     app.use('/node_modules/lit-html', express.static(litPath));
     app.use(express.static(path.join(__dirname, '../client')));
 
-    app.get('/dash.json', this.handleDashJson.bind(this));
+    app.get('/dash.json', new DashData(this.github).getHandler());
```

Comment:
Nice, I like the idea of using this pattern to separate every handler into its own file.